.circleci/config.yml

```diff
@@ -89,6 +89,18 @@ jobs:
             nox -f django_cloud_deploy/nox.py -s e2e_test
           # Timeout of e2e test is set to 30 minutes
           no_output_timeout: 30m
+  Coverage Report:
```

Comment:
You should add "Coverage Report" under "workflow". Otherwise you just defined this job and not running it.